bootstrapvz/plugins/puppet/tasks.py

```diff
@@ -1,14 +1,32 @@
 from bootstrapvz.base import Task
 from bootstrapvz.common import phases
-from bootstrapvz.common.tools import sed_i
+from bootstrapvz.common.tools import sed_i, log_check_call
 import os
+from docutils.nodes import description
+from distutils.log import info
+from bootstrapvz.common.releases import jessie, wheezy
 
 
+# This is an updated puppet module version, introducing the puppet 4 agent
+# and also enabling you to install puppet modules on the image ready.
+# For now it ASSUMES the following:
+# - Debian Wheezy Jessie install
+# - You want the puppet 4 agent (1.9.+) installed from apt.puppetlabs.com
+#
+# Modules isntalled with this plugin are "--force install"
+
+# TODO: plugin must be able to offer choice of distro package or 
```

Comment:
No TODOs in the code, you can create an issue with a todo list instead. This way people can comment on how to solve it etc.